plugins/woocommerce/package.json

```
@@ -542,7 +543,7 @@
 						"start": "env:start:blocks"
 					},
 					"events": [
-						"pull_request",
+						"temp_skip_pull_request",
```

Revert before merging

```suggestion
						"pull_request",
```